app/enterprise/1.3-x/brain-immunity/install-configure.md

```diff
@@ -3,207 +3,154 @@ title: Kong Brain & Kong Immunity Installation and Configuration Guide
-* Access to a platform for the Collector which has Docker installed. This system must be networked to “talk” to the Kong Enterprise system on which the Collector is installed.
-* Bintray access credentials (supplied by Kong) to access the downloads.
-
-#### Overview
+### Overview
+Kong Brain and Kong Immunity are installed as add-ons on Kong Enterprise, using a Collector App and a Collector Plugin to communicate with Kong Enterprise. The diagram illustrates how the Kong components work together, and are described below:
+* **Kong Enterprise**
+* **Kong Brain** (Brain) and/or **Kong Immunity** (Immunity) add-ons, according to your purchase.
+* **Collector App** enables communication between Kong Enterprise and Brain and/or Immunity. The Kong Collector App comes with your purchase of Bran and/or Immunity. 
+* **Collector Plugin** enables communication between Kong Enterprise and Kong Collector App.  The Kong Collector Plugin comes with your purchase of Brain and/or Immunity. 
 
-With the Collector, Kong Enterprise collects, coalesces and stores requests made through Kong. This plugin makes a copy of requests through Kong (not affecting the hot path) which are “Teed” to Kong Brain and/or Kong Immunity platforms. See the diagram below for a visual of how Kong Collector Plugin works with Kong Enterprise. 
+### Prerequisites
+Prerequisites for installing and configuring Brain and/or Immunity with Kong Enterprise include:
+* Kong Enterprise version 0.35.3+ or later, with a minimum of one Kong node and a working datastore (PostgreSQL).
+* Access to a platform for the Collector App with Docker installed. This system must be networked to communicate with the Kong Enterprise system where you have the Collector Plugin installed.
+* Access to Kong Bintray repository (https://bintray.com/kong), including your access credentials supplied by Kong. (BINTRAY_USERNAME, BINTRAY_API_KEY)
+* Docker compose file of your purchased version of Brain and/or Immunity, which display in Bintray as one of the following:
+   * kong/kong-brain-immunity-base
+   * kong/kong-brain-base
+   * kong/kong-immunity-base
+* Redis instance, which is included with Brain and/or Immunity. 
+* Swagger, which is included with Brain and/or Immunity.
 
-#### Configuration Order
+### Configure the Collector App and Collector Plugin
+To enable Kong Brain (Brain) and/or Kong Immunity (Immunity), you must first configure the Collector App and the Collector Plugin. This includes:
+* Deploying the Collector Plugin, which captures and sends traffic to the Collector App for data collection and processing.
+* Deploying the Collector App on your Docker aware platform.
+* Configuring the Collector Plugin and the Collector App to talk to each other. 
+* Testing the configuration to confirm everything is up and running. 
 
-Configure the Collector in the following order, with configuration steps provided below:
+Steps are:
+- [Step 1. Enable the Collector Plugin] (#Step-1.-Enable-the-Collector-Plugin)
```

Comment:
For internal linking to work, there cannot be a space between the `[]` and `()` and the link inside the parenthesis must match the header text, in URL format ie:
`- [Step 1. Enable the Collector Plugin](#step-1-set-up-the-collector-plugin)`